packages/rocketchat-api/server/v1/users.js

```diff
@@ -123,8 +124,24 @@ RocketChat.API.v1.addRoute('users.info', { authRequired: true }, {
 			return RocketChat.API.v1.failure(`Failed to get the user data for the userId of "${ username }".`);
 		}
 
+		user = result[0];
+		if (RocketChat.authz.hasPermission(this.userId, 'view-other-user-channels')) {
+			user.rooms = RocketChat.models.Subscriptions.findByUserId(this.userId, {
```

Comment:
this will add a lot of data to the response.. can we include this field only when requested (like with a `fields` param) ?